include/platform/mir/graphics/platform.h

```diff
@@ -163,8 +163,8 @@ class GLRenderingProvider : public RenderingProvider
 
     virtual auto as_texture(std::shared_ptr<Buffer> buffer) -> std::shared_ptr<gl::Texture> = 0;
 
-    virtual auto surface_for_output(
```

Comment:
Does this need to be renamed?

If it does, maybe it should be harmonised with `GLRenderingProvider::suitability_for_display`?

Reply:
Well, maybe I read it wrong, but it reads better to me:

```c++
    auto output_surface = chosen_allocator->surface_for_device(
        display_device, display_device.pixel_size(), *gl_config);
```

Reply:
Or, maybe this suggests another route? `DisplayOutput`?